src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs

```diff
@@ -169,7 +169,9 @@ public async Task UpdateAsync(TDocument document, Func<TDocument, Task> afterUpd
                 {
                     await DocumentStore.CancelAsync();
 
-                    throw new InvalidOperationException($"Can't update the '{typeof(TDocument).Name}' if not able to access the distributed cache");
+                    _logger.LogError("Can't update the '{DocumentName}' if not able to access the distributed cache", typeof(TDocument).Name);
+                    
+                    throw;
```

Comment:
Usually we don't log when we throw because when we throw it will be logged.

Reply:
This is based on your suggestion above ;)

Reply:
Which suggestion? Where?

Reply:
[Here](https://github.com/OrchardCMS/OrchardCore/pull/11881#discussion_r901043702)

Reply:
I re-read the comments, okay then ;)